scripts/clan.py

```diff
@@ -634,13 +634,13 @@ def load_clan_txt(self):
             members = sections[4].split(",")
             other_clans = []
         if len(general) == 9:
-            if general[3] == "None":
+            if general[3] == None or general[3] == "None":
```

Comment:
```suggestion
            if general[3] is None or general[3] == "None":
```
It's more Pythonic to say `if [variable] is None`, or alternatively, `if not [variable]`. Please make this change wherever relevant